### PR TITLE
Force Tracing : checkbox in query frontend to force a trace to be collected

### PR DESCRIPTION
**Pull request by Vanshikav123**
## Changes

A checkbox that can be enabled that will force tracing
#6311

### CODE REVIEW

**Comment by MichaHoffmann** [comment]
For UX: wouldnt it be nicer if we get a trace ID we can search for when this is forced? Is this already the case, i have not really tried out i must admit.

**Comment by yeya24** [comment]
> wouldnt it be nicer if we get a trace ID we can search for when this is forced

Let's keep the scope of this pr small and tackle this idea in another pr if nencessary

File: pkg/ui/react-app/src/pages/graph/Panel.tsx [hunk 1]
```diff
@@ -118,7 +120,8 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
     }
     this.handleEngine(this.props.options.engine);
 
-    this.handleChangeDeduplication = this.handleChangeDeduplication.bind(this);
+    this.handleChangeDeduplication = this.handleChangeDeduplication.bind(this); //forcetracing
+    this.handleChangeForceTracing = this.handleChangeForceTracing.bind(this);
```
Comment (yeya24) [inline pkg/ui/react-app/src/pages/graph/Panel.tsx:124]: You probably want to add `//forcetracing` comment this line

File: pkg/ui/react-app/src/pages/graph/Panel.tsx [hunk 2]
```diff
@@ -406,6 +420,14 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
               >
                 Use Partial Response
               </Checkbox>
+              <Checkbox
+                wrapperStyles={{ marginLeft: 20, display: 'inline-block' }}
+                id={`force-tracing-checkbox-${id}`}
+                onChange={this.handleChangeForceTracing}
+                defaultchecked={options.forceTracing}
+              >
+                Enable Force Tracing
```
Comment (yeya24) [inline pkg/ui/react-app/src/pages/graph/Panel.tsx:429]: Let's simply call it `Force Tracing`

**Review by yeya24 — COMMENTED** [review]
Tested locally and I think this works as expected

**Comment by Vanshikav123** [comment]
> > wouldnt it be nicer if we get a trace ID we can search for when this is forced
> 
> Let's keep the scope of this pr small and tackle this idea in another pr if nencessary

Should i start working on it?

**Comment by Vanshikav123** [comment]
Done !!

**Review by yeya24 — APPROVED** [review]

**Pull request merged**